cylc/uiserver/resolvers.py

```diff
@@ -534,24 +543,29 @@ async def service(
         info: 'ResolveInfo',
         command: str,
         workflows: Iterable['Tokens'],
-        kwargs: Dict[str, Any]
+        kwargs: Dict[str, Any],
     ) -> List[Union[bool, str]]:
-        if command == 'clean':
+
+        if command == 'clean': # noqa: <SIM116 Use a dictionary lookup instead of 3+ if/elif-statements> 
```

Comment:
Just the error code is fine 🙂
```suggestion
        if command == 'clean': # noqa: SIM116
```
(Otherwise you'll hit the "line too long" error code!)